rubocop-airbnb/config/rubocop-layout.yml

```diff
@@ -551,3 +551,6 @@ Layout/TrailingWhitespace:
   Description: Avoid trailing whitespace.
   StyleGuide: https://github.com/rubocop-hq/ruby-style-guide#no-trailing-whitespace
   Enabled: true
+
+Layout/IndentFirstParameter:
```

Comment:
It looks like the documented name is `Layout/FirstParameterIndentation`. Let's use that name instead of the alias to make it easier to find documentation.

https://github.com/rubocop-hq/rubocop/blob/a29694f7821ef51ca58528ca703bce9ec713716b/lib/rubocop/config_obsoletion.rb#L17